generators/client/templates/_package.json

```diff
@@ -8,6 +8,7 @@
     "<%= MAIN_SRC_DIR %>bower_components"
   ],
   "devDependencies": {
+    "bower": "1.7.9",
```

Comment:
he he..
Yes I know the trend. I myself have used npm instead of bower for the jhipster-react module. and we might move to full npm for JHipster 4 (I still hate npm and the way it works though)

I personally just hate the fact that npm downloads the internet again and again and again... so on a java project my intention was to keep it to a bare minimum hence we kept gulp in package.json as we write a gulpfile which works for a specific version and has dependent plugins were as bower is completely on its own and we dont do anything that would fail on a particular bower version, same goes for npm and node

so as you said lets agree to disagree.

I hope npm becomes something that we all can love :wink: